### PR TITLE
Adding system administration alias

### PR DESCRIPTION
**Pull request by ghost**
Adding a ducks alias

### CODE REVIEW

**Comment by MBuchaus** [comment]
Adding GRC aliases file for optional inclusion.. the ducks alias I've been using for years and would hope it's useful to others.

**Comment by MBuchaus** [comment]
Updated 2 other useful aliases

File: aliases/available/general.aliases.bash [hunk 1]
```diff
@@ -34,6 +34,15 @@ alias irc="$IRC_CLIENT"
...
 
+# find Space hogs
+alias ducks='du -cks * | sort -nr | head -11 |cut -f2- | xargs du -hs'
+
+# Set VI Mode
+alias sovi='set -o vi'
+
+# Sudo to root keeping your user environment
+alias rsu='sudo -E su -p'
+
```
Comment (edubxb) [inline aliases/available/general.aliases.bash]: `rsu` can be defined as:

`sudo -E -s`

I think, no need to use sudo an su to achieve this, but not sure if this works in OS X.

**Comment by MBuchaus** [comment]
the second is to pull the the user env when you su to root.  (vi - shell - etc)

**Comment by MBuchaus** [comment]
the comment should read sudo ( SU ) to root user. as a substitute for su -

**Comment by NoahGorny** [comment]
Hey, seems like this PR has gone stale. So I am closing it. Thanks for contributing!

**Pull request closed without merging**